examples/ssr/README.md

```diff
@@ -0,0 +1,9 @@
+# Server Side Rendering Example
+
+This example takes our basic example and adds Server Side Rendering to it.
```

Comment:
Let's enhance this with:
- custom title in the HTML page: React Router - SSR Example
- points of interest about this example
- server.js
- two entry points (one for client, one for server)
- where to start? maybe use the `?file` param in the stackblitz link too